### ISSUE TITLE
Problem with DeepExplainer. Getting Error when passing background images.

### ISSUE THREAD

**Issue by mehrdadasadiut** [shap/shap#2759]
Hello.
Here is my pre-trained model summary. I get errors when using DeepExplainer.  I am using 503 test backgrounds for computing Shap values.
Anyone can give a hand? Thanks.
Error : Layer "sequential_1" expects 1 input(s), but it received 503 input tensors. Inputs received: [<tf.Tensor: shape=(1, 300, 300, 3), dtype=float64, numpy=

Model Summary: Model: "sequential_1"
_________________________________________________________________
 Layer (type)                Output Shape              Param #   
=================================================================
 conv2d_3 (Conv2D)           (None, 300, 300, 32)      896       
                                                                 
 max_pooling2d_3 (MaxPooling  (None, 150, 150, 32)     0         
 2D)                                                             
                                                                 
 conv2d_4 (Conv2D)           (None, 150, 150, 64)      18496     
                                                                 
 max_pooling2d_4 (MaxPooling  (None, 75, 75, 64)       0         
 2D)                                                             
                                                                 
 conv2d_5 (Conv2D)           (None, 75, 75, 32)        18464     
                                                                 
 max_pooling2d_5 (MaxPooling  (None, 37, 37, 32)       0         
 2D)                                                             
                                                                 
 flatten_1 (Flatten)         (None, 43808)             0         
                                                                 
 dense_3 (Dense)             (None, 64)                2803776   
                                                                 
 dropout_2 (Dropout)         (None, 64)                0         
                                                                 
 dense_4 (Dense)             (None, 32)                2080      
                                                                 
 dropout_3 (Dropout)         (None, 32)                0         
                                                                 
 dense_5 (Dense)             (None, 6)                 198       
                                                                 
=================================================================
Total params: 2,843,910
Trainable params: 2,843,910
Non-trainable params: 0
